tests/test-timber-menu.php

```diff
@@ -145,20 +145,56 @@ function testMenuTwigWithClasses() {
 	function testMenuItemLink() {
 		self::setPermalinkStructure();
 		self::_createTestMenu();
-		$menu = new TimberMenu();
+		$menu = new Timber\Menu();
 		$nav_menu = wp_nav_menu( array( 'echo' => false ) );
 		$this->assertGreaterThanOrEqual( 3, count( $menu->get_items() ) );
 		$items = $menu->get_items();
 		$item = $items[1];
-		$this->assertTrue( $item->external() );
+		$this->assertTrue( $item->is_external() );
 		$struc = get_option( 'permalink_structure' );
 		$this->assertEquals( 'http://upstatement.com', $item->url );
 		$this->assertEquals( 'http://upstatement.com', $item->link() );
 	}
 
+	function testMenuItemIsTargetBlank() {
+		self::_createTestMenu();
+		$menu = new Timber\Menu();
+		$items = $menu->get_items();
+
+		// Menu item without _menu_item_target set
+		$item = $items[0];
+		$this->assertFalse( $item->is_target_blank() );
+
+		// Menu item with _menu_item_target set to '_blank'
+		$item = $items[1];
+		$this->assertTrue( $item->is_target_blank() );
+
+		// Menu item with _menu_item_target set to ''
+		$item = $items[2];
+		$this->assertFalse( $item->is_target_blank() );
+	}
+
+	function testMenuItemTarget() {
+		self::_createTestMenu();
+		$menu = new Timber\Menu();
+		$items = $menu->get_items();
+
+		// Menu item without _menu_item_target set
+		$item = $items[0];
+		$this->assertEquals( '_self', $item->target() );
+
+		// Menu item with _menu_item_target set to '_blank'
+		$item = $items[1];
+		$this->assertEquals( '_blank', $item->target() );
+
+		// Menu item with _menu_item_target set to ''
+		$item = $items[2];
+		$this->assertFalse( '_self', $item->target() );
```

Comment:
@gchtr wanted to check on this test (it's failing right now). Should `$item->target()` register as `_self` or `false` in this scenario?

Reply:
@jarednova See #2019.